source/code/plugins/filter_syslog.rb

```diff
@@ -30,6 +30,7 @@ def filter(tag, time, record)
       # Use Time.now, because it is the only way to get subsecond precision in version 0.12.
       # The time may be slightly in the future from the ingestion time.
       record["Timestamp"] = OMS::Common.format_time(Time.now.to_f)
+      record["EventTime"] = OMS::Common.format_time(time)
```

Comment:
The reason we use Time.now.to_f is that if two identical syslog events happen in the same second, ODS will only show one. Also, different events within the same second would appear out of order in the OMS portal...  @robbiezhang could give more details but sub second precision is very important. I'm not signing off